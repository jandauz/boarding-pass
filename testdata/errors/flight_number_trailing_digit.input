M1DESMARAIS/LUC       EABC123 YULFRAAC 08340326J001A0025 100