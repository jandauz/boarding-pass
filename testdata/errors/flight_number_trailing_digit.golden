bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAIS/LUC       EABC123 YULFRAAC 08340326J001A0025 100"
  |                                         ^ got "08340"
  |
  = reason: data for "Flight Number" must be 4 digits with leading zeroes followed by an optional alpha suffix or whitespace
